v2/AdminCore.Services/EventService.cs

```diff
@@ -355,14 +345,10 @@ private void ValidateRemainingHolidaysAndUpdate(Event eventToUpdate)
 
     private EventDto ValidateRemainingHolidaysAndCreate(Event newEvent)
     {
-      if (EmployeeHasEnoughHolidays(newEvent))
-      {
-        DatabaseContext.EventRepository.Insert(newEvent);
-        DatabaseContext.SaveChanges();
-        return _mapper.Map<EventDto>(newEvent);
-      }
-
-      throw new Exception("Not enough holidays to book");
+      if (!EmployeeHasEnoughHolidays(newEvent)) throw new Exception("Not enough holidays to book");
```

Comment:
Try not to keep if statements on 1 line like that. Much cleaner to read like:

if (!EmployeeHasEnoughHolidays(newEvent)) 
{
    throw new Exception("Not enough holidays to book");
}